builtin/providers/aws/resource_aws_rds_cluster_instance_test.go

```diff
@@ -187,6 +187,25 @@ func testAccCheckAWSClusterInstanceExists(n string, v *rds.DBInstance) resource.
 	}
 }
 
+func testAccAWSCluster_with_InstanceEnhancedMonitor(t *testing.T) {
```

Comment:
Can you rename this to be `TestAccAWSRDSCluster_withInstanceEnhancedMonitor` so that a single call will run all tests?